automation/taskcluster/lib/tasks.py

```diff
@@ -161,6 +163,50 @@ def craft_test_task(self, variant):
             },
         )
 
+   
+    def craft_ui_tests_task(self):
+        artifacts = {
+            "public": {
+                "type": "directory",
+                "path": "/build/fenix/results",
+                "expires": taskcluster.stringDate(taskcluster.fromNow(DEFAULT_EXPIRES_IN))
+            }
+        }
+
+        env_vars = {
+            "GOOGLE_PROJECT": "moz-fenix",
+            "GOOGLE_APPLICATION_CREDENTIALS": ".firebase_token.json"
+        }
+
+        gradle_commands = (
+            './gradlew --no-daemon clean assembleArmDebug assembleArmDebugAndroidTest',
+        )
+
+        test_commands = (
+
+            'chmod +x automation/taskcluster/androidTest/ui-test.sh',
```

Comment:
nit: I think if you commit this file with `+x` then you do not need to mark it as executable here every time? 🤔

Reply:
👍